packages/files/_test.pony

```diff
@@ -473,7 +474,7 @@ class iso _TestFileCreateExistsNotWriteable is _NonRootTest
       mode.owner_read = true
       mode.owner_write = true // required on Windows to delete the file
       filepath.chmod(mode)
-      filepath.remove()
+      h.assert_true(filepath.remove())
```

Comment:
seems like this test should be updated to match _TestFileRemoveReadOnly in terms of the try/then and asserting both chmod and remove.

Reply:
I've updated it a bit.